frontend/src/pages/chat/components/chat-header/styles.module.scss

```diff
@@ -0,0 +1,31 @@
+.chat-header {
+  position: sticky;
+  top: calc(0%);
+  width: 100%;
+  background-color: var(--white);
+  box-shadow: 1px 15px 15px -15px var(--white);
+}
+
+.header-content {
+  display: flex;
+  align-items: center;
+  padding: 24.5px 36px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.title {
+  margin: 0 16px;
+  font-weight: 600;
+  font-size: 20px;
+  font-family: Montserrat, sans-serif;
```

Comment:
There's already setted global font on `body` in `scaffolding.scss`, so you don't have to write it anywhere) It's not the only one part of code with it

Reply:
All done!